workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -64,21 +64,20 @@ def fig_to_b64(fig, format="png", dpi=None):
 """
 )
 
-CBAR_RANGE = {
-    "eastward_wind_pressure_level_zonal_bias": 30,
-    "northward_wind_pressure_level_zonal_bias": 3,
-    "air_temperature_pressure_level_zonal_bias": 15,
-    "specific_humidity_pressure_level_zonal_bias": 1e-3,
-    "vertical_wind_pressure_level_zonal_bias": 0.02,
-    "mass_streamfunction_pressure_level_zonal_bias": 100,
+CONTOUR_LEVELS = {
```

Comment:
Thanks, optional but now that the levels are all centered about zero it might be nice to add a test that all items in this dict have that property (since it may not be obvious to someone adding a new key that the aranges are centered like this).

Reply:
I think we might not want to enforce this in case someone adds hard-coded contour levels for variables that are not biases (and so not necessarily centered at zero).